packages/driver/src/cy/commands/local_storage.ts

```diff
@@ -3,6 +3,8 @@ import _ from 'lodash'
 import $errUtils from '../../cypress/error_utils'
 import $LocalStorage from '../../cypress/local_storage'
```

Comment:
I would have expected `../../cypress/local_storage` to no longer be needed after this change.

Reply:
This PR is meant to be purely additive. Any refactoring is out of scope and should be separate.